attribute/helpers_for_test.go

```diff
@@ -0,0 +1,23 @@
+package attribute
```

Comment:
I think various parts of our tooling are looking for test files that end in `...test.go`, so it might be better to rename this to something like `test_helper.go`, so there's no potential for it to get confused for a test class, what do you think?

Reply:
This is actually depending on that behavior - files ending in _test.go aren't included in `go build` the package and are only compiled for `go test`

Would helper_test.go be less obtuse? (Go doesn't really have test classes)

Reply:
Yes helper_test.go is better I think 👍